nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -1922,6 +1922,12 @@
           generating host-global NNCP configuration.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          The <link xlink:href="https://musikcube.com">musikcube</link>
```

Comment:
We don't need to list every minor package update between versions in release notes. Please drop this.

Reply:
Got it, thanks @aanderse.